api.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const ncNewsApi = axios.create({
+  baseURL: "https://nc-news-be-2xk8.onrender.com/api",
+});
+
+// RETRIEVE ALL ARTICLES
+export const getAllArticles = () => {
+  return ncNewsApi.get("/articles?limit=5").then(({ data }) => {
```

Comment:
not sure why you're adding the limit here, which also isn't working as more than 5 are displaying